packages/storybook/cxl-ui/cxl-playbook-accordion/theme=cxl-playbook.faq.stories.js

```diff
@@ -0,0 +1,55 @@
+import { html } from 'lit-html';
+import '@conversionxl/cxl-ui/src/components/cxl-app-layout.js';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+import faqData from '../cxl-vaadin-accordion/theme=cxl-faq.data.json';
+import '@vaadin/vaadin-ordered-layout';
+
+export default {
+  title: 'CXL UI/cxl-playbook-accordion',
+};
+
+export const CXLPlaybookAccordionThemeFaq = () => html`
+  <style>
+    .plural .entry-title {
+      margin: 0;
+    }
+    cxl-playbook-accordion {
+      padding: 10px;
```

Comment:
No magic number units allowed.

Why needed, even?

Reply:
> No magic number units allowed.
 
I didn't know that this refers to the `stories` which are just examples of usage.
Removed in: f3ef028